operate/schema/src/main/java/io/camunda/operate/schema/SchemaStartup.java

```diff
@@ -63,18 +59,19 @@ public void initializeSchema() throws MigrationException {
             "SchemaStartup: schema won't be created, it either already exist, or schema creation is disabled in configuration.");
       }
 
-      if (!newFields.isEmpty()) {
-        if (createSchema) {
-          schemaManager.updateSchema(newFields);
-        } else {
-          LOGGER.info(
-              "SchemaStartup: schema won't be updated as schema creation is disabled in configuration.");
-        }
-      }
-      if (migrationProperties.isMigrationEnabled()) {
-        LOGGER.info("SchemaStartup: migrate schema.");
-        migrator.migrateData();
-      }
+      //      if (!newFields.isEmpty()) {
+      //        if (createSchema) {
+      //          schemaManager.updateSchema(newFields);
+      //        } else {
+      //          LOGGER.info(
+      //              "SchemaStartup: schema won't be updated as schema creation is disabled in
+      // configuration.");
+      //        }
+      //      }
+      //      if (migrationProperties.isMigrationEnabled()) {
+      //        LOGGER.info("SchemaStartup: migrate schema.");
+      //        migrator.migrateData();
+      //      }
       LOGGER.info("SchemaStartup finished.");
```

Comment:
❓ why leaving out commented code? why not deleting it?

Reply:
I have a thought that when we tackle full deletion of migration code from Operate, this can give us good entry point to search for it. But you're right, I will delete it.